stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -38,6 +51,8 @@ internal class PaymentOptionsViewModel(
             eventReporter.onSelectPaymentOption(paymentSelection)
             prefsRepository.savePaymentSelection(paymentSelection)
             _userSelection.value = paymentSelection
+            // Update the returned value with the savedCard rather than the NewCard
+            // so that we don't jump the next time.
```

Comment:
What line does this comment apply to? At first glance it looks like a "todo" comment.

Reply:
yeah it is a todo for when I get to actually saving the card :D.